server/models/User.py

```diff
@@ -97,9 +108,21 @@ class UserSchema(Schema):
     latitude = fields.Float()
     longitude = fields.Float()
     formattedAddress = fields.String()
```

Comment:
Are we just going to change this when this gets merged? We plan to store address, and no longer the smaller address components, right?

Reply:
Yes, i left that code alone because your branch already takes care of that. so it will get changed when your pr gets merged into dev.